packages/offix-datastore/src/Model.ts

```diff
@@ -38,20 +38,6 @@ export interface ModelConfig<T = unknown> {
    * Model fields
    */
   fields: Fields<T>;
-
-  /**
-   * Delta Query and Subscription filter
-   */
-  predicate?: Predicate<T>;
```

Comment:
Model should never actually have those as it mixing concerns - model represents local state and replication information should be passed separately.